agent/api/task_unix.go

```diff
@@ -15,16 +15,126 @@
 
 package api
 
+import (
+	"path/filepath"
+
+	"github.com/aws/amazon-ecs-agent/agent/config"
+	"github.com/aws/amazon-ecs-agent/agent/utils"
+	"github.com/cihub/seelog"
+	docker "github.com/fsouza/go-dockerclient"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/pkg/errors"
+)
+
 const (
 	portBindingHostIP = "0.0.0.0"
+	maxTaskVCPULimit  = 10     // 10 VCPUs limit
+	defaultCPUPeriod  = 100000 // 100ms
```

Comment:
Can you set this const with the appropriate unit (`100 * time.Millisecond`) https://play.golang.org/p/NpNlwVp4ez. You can also look at https://golang.org/pkg/time/#Duration for a reference of how to do various conversion you may need.

Reply:
Ok